api/lib/domain/usecases/retrieve-last-or-create-certification-course.js

```diff
@@ -76,7 +77,7 @@ const retrieveLastOrCreateCertificationCourse = async function ({
     const isUserLanguageValid = _validateUserLanguage(languageService, user.lang);
 
     if (!isUserLanguageValid) {
-      throw new Error(`Cant create a certification course in ${user.lang}`);
+      throw new LanguageNotSupportedError();
```

Comment:
Ajouter un paramètre pour plus d'info
```suggestion
      throw new LanguageNotSupportedError(user.lang);
```